test/IntegrationTests/Helpers/MockZipkinCollector.cs

```diff
@@ -122,6 +125,7 @@ public static async Task<MockZipkinCollector> Start(ITestOutputHelper output, st
 
             await Task.Delay(500);
         }
+        while (DateTime.Now < deadline);
```

Comment:
I know that it is test code and not related to the PR, but it is good practice to work on UTC time when compare it with something else.

Similar change is needed for deadline definition.

```suggestion
        while (DateTime.UtcNow < deadline);
```

Reply:
This would have to be also changed: `var deadline = DateTime.Now.Add(timeout.Value);`

Reply:
Yeah, I missed that.